reference/conanfile/methods/config_options.rst

```diff
@@ -3,17 +3,48 @@
 config_options()
 ================
 
-The ``config_options()`` method
-is used to configure or constraint the available options in a package, **before** they are given a value. A typical use case is to remove an option in a given platform. For example,
-the ``fPIC`` flag doesn't exist in Windows, so it should be removed in this method like so:
+The ``config_options()`` method is used to configure or constraint the available options in a package, **before** they are given a value.
+A typical use case is to remove an option in a given platform.
+For example, the ``SSE2`` flag doesn't exist in architectures different than 32 bits, so it should be removed in this method like so:
 
 .. code-block:: python
 
     def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
+        if self.settings.arch != "x86_64":
+            del self.options.with_sse2
 
-The ``config_options()`` method executes before the ``configure()`` method, and before the actual assignment of the ``options`` values, but after settings are already defined.
+The ``config_options()`` method executes before the ``configure()`` method, and before the actual assignment of the ``options`` values,
+but after settings are already defined.
```

Comment:
I think we could rephrase it to make it easier to read:

```
The ``config_options()`` method executes:
* Before calling the ``configure()`` method.
* Before assigning the ``options`` values.
* After ``settings`` are already defined.
```
WDYT?

Reply:
I did not change the wording around this, just split the line to make it easier to read in the editor. Anyway, I can add your suggestion

Reply:
done